ServiceWebsite/ServiceWebsite/Controllers/ParticipantController.cs

```diff
@@ -34,8 +42,75 @@ public ParticipantController()
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         public async Task<IActionResult> UpdateSuitabilityAnswers(Guid hearingId, [FromBody]List<HearingSuitabilityAnswer> answers)
         {
-          await Task.Yield();
-          return NoContent();
+            if (hearingId == Guid.Empty)
+            {
+                return new BadRequestObjectResult($"Please provide a valid {nameof(hearingId)}");
+            }
+            try
+            {
+                var participantId = await _hearingService.GetParticipantIdAsync(User.Identity.Name, hearingId);
+
+                if (!participantId.HasValue || participantId == Guid.Empty)
+                {
+                    return new UnauthorizedObjectResult($"User is not a participant of hearing with id '{hearingId}'");
+                }
+                
+                if (answers.Count == 0)
```

Comment:
You could do this before you get the participant id to save a call to the server in case the input data is wrong.

Reply:
Resolved